dockers/docker-sonic-telemetry/telemetry_vars.j2

```diff
@@ -0,0 +1,5 @@
+{
+    "x509" : "{% if DEVICE_METADATA['x509'] is defined %}{{ DEVICE_METADATA['x509'] }}{% endif %}",
```

Comment:
TELEMETRY key is preferred place to load cert path. Why is it not in the template file?

Reply:
Both certs and x509 are maps that contains keys. Here is a sample of the output from my dut:
```
{
    "x509" : "",
    "gnmi" : {"client_auth": "true", "log_level": "2", "port": "50051"},
    "certs": {"ca_crt": "/etc/sonic/telemetry/dsmsroot.cer", "server_key": "/etc/sonic/telemetry/streamingtelemetryserver.key", "server_crt": "/etc/sonic/
}
```

Reply:
@hui-ma there is no change in between existing script and the new one. The change is that all calls to sonic-cfggen are bundled in one call using templates. Some of the template var are maps that contains further keys/vars.